ckanext/chevrontheme/templates/snippets/package_item.html

```diff
@@ -21,14 +21,18 @@
 {% block package_item %}
   <tr>
     <td>
-      {{ h.link_to(package.chevron_dataset or 'Unknown', h.url_for(controller='package', action='read', id=package.name)) }}
```

Comment:
this change needs to be conditional too... and I'm pretty sure it's an incorrect change anyway. As I mentioned the other day, the ingest for PaDIL should be setting 'chevron_dataset' to 'PaDIL' (or whatever) on all the packages. This may require a change in chevron-ingest and a re-generation of the schema.